examples/workflow/aio/context_interceptors_example.py

```diff
@@ -0,0 +1,152 @@
+    ) -> Any:
+        return nxt(
+            CallChildWorkflowInput(
+                workflow_name=input.workflow_name,
+                args=_merge_ctx(input.args),
+                instance_id=input.instance_id,
+                workflow_ctx=input.workflow_ctx,
+                metadata=input.metadata,
+                local_context=input.local_context,
+            )
+        )
+
+    def call_activity(
+        self, input: CallActivityInput, nxt: Callable[[CallActivityInput], Any]
+    ) -> Any:
+        return nxt(
+            CallActivityInput(
+                activity_name=input.activity_name,
+                args=_merge_ctx(input.args),
+                retry_policy=input.retry_policy,
+                workflow_ctx=input.workflow_ctx,
+                metadata=input.metadata,
+                local_context=input.local_context,
+            )
+        )
+
+
+class ContextRuntimeInterceptor(BaseRuntimeInterceptor):
+    def execute_workflow(
+        self, input: ExecuteWorkflowInput, nxt: Callable[[ExecuteWorkflowInput], Any]
+    ) -> Any:  # type: ignore[override]
+        if isinstance(input.input, dict) and 'context' in input.input:
+            set_ctx(input.input['context'])
+        try:
+            return nxt(input)
+        finally:
+            set_ctx(None)
+
+    def execute_activity(
+        self, input: ExecuteActivityInput, nxt: Callable[[ExecuteActivityInput], Any]
+    ) -> Any:  # type: ignore[override]
+        if isinstance(input.input, dict) and 'context' in input.input:
+            set_ctx(input.input['context'])
+        try:
+            return nxt(input)
+        finally:
+            set_ctx(None)
+
+
+# Example workflow and activity
+def activity_log(ctx, data: dict[str, Any]) -> str:  # noqa: ANN001 (example)
+    # Access restored context inside activity via contextvars
+    return f'ok:{get_ctx()}'
+
+
+def workflow_example(ctx, x: int):  # noqa: ANN001 (example)
+    y = yield ctx.call_activity(activity_log, input={'msg': 'hello'})
+    return y
+
+
+def wire_up() -> tuple[WorkflowRuntime, DaprWorkflowClient]:
+    runtime = WorkflowRuntime(
+        runtime_interceptors=[ContextRuntimeInterceptor()],
+        workflow_outbound_interceptors=[ContextWorkflowOutboundInterceptor()],
+    )
+    client = DaprWorkflowClient(interceptors=[ContextClientInterceptor()])
+
+    # Register workflow/activity
+    runtime.workflow(name='example')(workflow_example)
+    runtime.activity(name='activity_log')(activity_log)
+    return runtime, client
+
+
+if __name__ == '__main__':
+    # This section demonstrates how you would set a context and schedule a workflow.
+    # Requires a running Dapr sidecar to actually execute.
+    rt, cli = wire_up()
+    set_ctx({'tenant': 'acme', 'request_id': 'r-123'})
+    # instance_id = cli.schedule_new_workflow(workflow_example, input={'x': 1})
+    # print('scheduled:', instance_id)
+    # rt.start(); rt.wait_for_ready(); ...
```

Comment:
can you uncomment these so users can run and see the context injected. Do we print out the stuff that gets injected so this example is crystal clear to the value add of the interceptor from an end user POV?